miscellaneous-commands/userconfig.md

```diff
@@ -22,8 +22,9 @@ For more information on editing permission requirements for specific users/roles
 /userconfig settings [reaction-role-notifications] [vote-reminders]
 ```
 
-* `reaction-role-notifications`: <TODO: describe>
-* `vote-reminders`: <TODO: describe>
+* `reaction-role-notifications`: This subcommand lets you enable or disable notifications when you add or remove [a reaction role](/start-up/setting-up-reaction-roles.md) from yourself.
+* `vote-reminders`: This subcommand will remind you daily to [vote for MoonlightBot.](/MoonlightBot-docs/support/upvote-moonlightbot.md)
+* `locale`: This subcommand lets you choose what language you want your notifications to be in. For example, if you live in Canada, you can set the locale to French or English. If you'd rather have the bot automatically determine the locale, you can set it to "Auto" and it will automatically determine the locale.
```

Comment:
not only notifications, pretty much all of the bot's responses and interactions with you. the canada example is not a good idea, redo it

autodetermines from the language set on the discord app when you use a command